p2p/simulations/p2ptest.go

```diff
@@ -64,29 +68,66 @@ func (p *myProtocol) sendMyMessage() {
 
 type Config struct {
 	P2PConfig    p2p.Config
+	StaticNodes  []string
 	RemoteNodeID string // optional. peer node
 	RemoteAddr   string // optional, format 182.87.223.29:39008
 }
 
+func myresolve(id string) (*discovery.Node, error) {
+	nodeHeader := "snode://"
+	id = id[len(nodeHeader):]
+
+	idSplit := strings.Split(id, "@")
+	if len(idSplit) != 2 {
+		return nil, errors.New("invalidNodeError")
+	}
+
+	address, err := hex.DecodeString(idSplit[0])
+	if err != nil {
+		return nil, err
+	}
+
+	publicKey, err := common.NewAddress(address)
+	if err != nil {
+		return nil, err
+	}
+
+	addr, err := net.ResolveUDPAddr("udp", idSplit[1])
+	if err != nil {
+		return nil, err
+	}
+
+	node := discovery.NewNodeWithAddr(publicKey, addr)
+	return node, nil
+}
+
 func main() {
 	var config *Config = new(Config)
 	_, err := toml.DecodeFile("test.toml", config)
 	if err != nil {
 		fmt.Println(err)
 		return
 	}
+	//fmt.Println(config.StaticNodes[0])
+	////os.Exit(0)
 	// no config check
 	myServer := &p2p.Server{
 		Config: config.P2PConfig,
 	}
 
-	if config.RemoteNodeID == "" {
+	if len(config.StaticNodes) == 0 {
 		fmt.Println("No remote peer configed, so is a static peer")
 	} else {
-		nodeIDPeer := common.HexToAddress(config.RemoteNodeID)
-		addrPeer, _ := net.ResolveUDPAddr("udp4", config.RemoteAddr)
-		nodeObjPeer := discovery.NewNodeWithAddr(nodeIDPeer, addrPeer)
-		myServer.StaticNodes = append(myServer.StaticNodes, nodeObjPeer)
+		for _, id := range config.StaticNodes {
+
+			n, err := discovery.NewNodeFromString(id)
+			//n, err := myresolve(id)
+			if err != nil {
+				fmt.Println(err)
```

Comment:
log instead of fmt.println. For cmd, we could add console listener for the log system for stdout purpose.

Reply:
p2ptest is a test console program, do not need this